metadata, static files & more

opanchen opened the pull request:
- створено початкову версію метаданих, готову до оновлення та  подальшої оптимізації під час масштабування проекту (в якості тимчасового ogp експортовано дизайн головної секції, url в env додати після деплою - для розробки використано фейковий);
- оновлено Husky для коректної роботи на Mac;
- змінено значення атрибута lang в лояуті на коректний;
- експортовано лого та фонові зображення на головну секцію з макету:
- створено компонент порталу, початковий компонент модального вікна (в розробці), та додатковий TestClientComponent.tsx (згодом видалиться);

opanchen committed:
add initial metadata

opanchen committed:
upd husky

Add additional config via command line to be able to use Husky correctly on Mac.

opanchen committed:
fix html lang attribute value

opanchen committed:
export logo & hero bg-images

opanchen committed:
create Portal & init Modal

nadiakhoptiuk commented on public/images/backgrounds/hero/hero-mobile.webp:
фон для мобільного на 360 буде малим на екранах 480

opanchen replied:
думаю, це вже стосується реалізації фону на компоненті - задача стояла викачати зображення з макету
далі вже розробник підбере зображення під конкретний брейкпоінт і css-правило (наприклад, відразу з таблетної версії)

nadiakhoptiuk commented on public/images/backgrounds/hero/hero-desktop.webp:
для десктопа фон викачаний без градієнта

opanchen replied:
для десктопу в макеті зображення не має градієнту (його додано окремим шаром), на інших екранах дизайнер відразу додав обрізані забраження з суцільним градієнтом

https://prnt.sc/wezMq63dn9nS

https://prnt.sc/hDHMW1qB0VCI

nadiakhoptiuk commented on src/data/meta.json:
```diff
@@ -0,0 +1,67 @@
+    "images": [
+      {
+        "url": "/meta/ogp-image-demo.jpg",
+        "width": 1444,
```
сама картинка має 2880 пікселів у ширину

opanchen replied:
виправлено згідно розмірів файла:
        "width": 2880,
        "height": 1600,

nadiakhoptiuk commented on tailwind.config.ts:
```diff
@@ -46,6 +46,7 @@ const config: Config = {
           heading: "#1A341B",
           text: "#4A4742",
           desc: "#8B8B8B",
+          backdrop: "rgba(0, 0, 0, 0.8)",
```
тут в content вище зайва папка "./src/pages/**/*.{js,ts,jsx,tsx,mdx}"

opanchen replied:
видалено

nadiakhoptiuk commented on src/components/ui/index.ts:
як показала практика, реекспорти через один файл можуть неприємно вистрілити (зациклити додаток або видавати помилки, які неможливо пофіксити іншим методом, окрім як експортувати окремо)
те саме у sections

opanchen replied:
видалено

opanchen committed:
fix tailwind-config, metadata, etc.

Fix tailwind.config.ts. Fix sizes for ogp-image according to file size. Remove additional re-export from base & ui components folders.
